profile_library/signify/LCG002/model.json

```diff
@@ -22,7 +22,10 @@
     "5060830P7_04",
     "5060831P7_02",
     "5060831P7_03",
-    "5060831P7_04"
+    "5060831P7_04",
+    "5047231P6",
+    "5047331P6",
+    "5047431P6"
```

Comment:
These seem to be different lights. 
5060830P7 is hue centris light with 3 spots.

5047231P6, 5047331P6 and 5047431P6 are Hue Buckram lights with 2, 3 or 4 spots.
So we cannot use this profile as it's measured with Hue Centris 3 spots, so power consumption will be diffent for these spots.

Reply:
They are all single spots, but in hue buckram "chassis". I'm not sure why the bulbs are reporting as different versions. Maybe for the hue app to know the difference?

I have multiple buckram's, some containing 2, 3 or 4 spots. I see each spots as different entities in the UI (I have never seen them together). They got those numbers.

Am I missunderstanding something?

Reply:
You mean you have 3 separate light entities in HA to control a 3 spot buckram fixture?
An for the 4 spot fixture you have 4 entities?

Could you show screenshot of the 5047431P6 device for example?

And how are we certain this is exactly the same spot and circuitry as the Cendris light? Could probably be, but how can we be certain?

Reply:
Here is a screenshot of 3 different lamps, with 3, 2 and 4 bulbs. I installed them mostly at the same time, so the missmatch of different bulbs is probably me mixing them when installing them.
<img width="859" alt="Screenshot 2024-10-21 at 22 17 45" src="https://github.com/user-attachments/assets/140ca9f5-b359-41b3-b19f-9a72f8ab42a7">

When I searched for them in at https://library.powercalc.nl/, I did get hits on LCG002. Thats why I assumed it was the correct bulb (the spec looked plausible as well). They look the same, but others are also differenting on buckram and normal spots. Like at https://github.com/JohanBendz/com.philips.hue.zigbee/blob/8827cf8e3f698faac044ffc8f9fb05f29aa51d0e/drivers/5047131P6/driver.compose.json

Anything I can look for physically on them maybe? Or, if it's too uncertain, maybe it's best to add it as a custom profile instead?

Reply:
Could probably be they are the same GU10 spot and same circuitry. But I'm not 100% certain and its important for other also to have correct data in de database.
Do you have any possibilities to do measurements?
https://docs.powercalc.nl/contributing/measure/

Reply:
That might take some time to figure out. I'll look around for the equipment needed and come back to this issue if I ever find something useful that doesnt cost that much. These lamps are attached in ways I can't easiely measure, but maybe there exists a little standalone lamp that I can attach the spot to directly. 🤔

In the meantime. I tried to add the changed models file to `powercalc/profiles/signify/LCG002/model.json` locally. It didnt do anything to powercalc :(

Reply:
You can just use a GU10 fitting or converter for normal light bulb:
- https://www.amazon.nl/Evolution-Lamphouder-Halogeenlampen-Inbouwframe-Keramische/dp/B00K2THKH0/ (you'll find these separate for only few bucks on the internet)
- https://www.amazon.nl/Miratic-Porselein-metalen-Gloeilampadapter-Gloeilamp-GU10/dp/B0CYM4Q1ZG/ref=sr_1_23_sspa

You don't have to copy the LCG002 profile, you can just select it when setting up manually using these instructions:
https://docs.powercalc.nl/sensor-types/virtual-power-library/